src/core/components/SearchModal.module.css

```diff
@@ -0,0 +1,146 @@
+.container {
```

Comment:
We're working on doing the css from a mobile-first approach. Let's change this file to follow similar setup.

Essentially means making all the base classes the mobile stuff and tucking desktop classes inside

```
@media (max-width: 768px) {
  
}
```